packages/custom-decorators/test/is-enum-decorator.test.ts

```diff
@@ -0,0 +1,79 @@
+import { IsEnum } from '../src/is-enum.decorator';
+import { Validator, ValidatorOptions } from 'class-validator';
+
+function validateValues(
+  object: { testProperty: any },
+  values: any[],
+  validatorOptions?: ValidatorOptions,
+): Promise<any> {
+  const validator = new Validator();
+  const promises = values.map((value) => {
+    object.testProperty = value;
+    return validator.validate(object, validatorOptions).then((errors) => {
+      expect(errors.length).toEqual(0);
+      if (errors.length > 0) {
+        console.log(`Unexpected errors: ${JSON.stringify(errors)}`);
+        throw new Error('Unexpected validation errors');
+      }
+    });
+  });
+
+  return Promise.all(promises);
+}
+
+function checkReturnedError(
+  object: { testProperty: any },
+  values: any[],
+  validationType: string,
+  messages: string[],
+  validatorOptions?: ValidatorOptions,
+): Promise<any> {
+  let messagesIncrementor: number = 0;
+  const validator = new Validator();
+  const promises = values.map((value) => {
+    object.testProperty = value;
+    return validator.validate(object, validatorOptions).then((errors) => {
+      expect(errors.length).toEqual(1);
+      expect(errors[0].target).toEqual(object);
+      expect(errors[0].property).toEqual('testProperty');
+      expect(errors[0].constraints).toEqual({ [validationType]: messages[messagesIncrementor] });
+      expect(errors[0].value).toEqual(value);
+      messagesIncrementor++;
+    });
+  });
+
+  return Promise.all(promises);
+}
```

Comment:
In my opinion, those utility function adds more complexity than they solve. And both are used once, so they didn't reduce duplication.

Do you think they are useful? 🤔